### DISCUSSION TITLE
What subtitle formats might we lack support for?

### DISCUSSION THREAD

**Discussion by chenlung** [URUWorks/TeroSubtitler#50] [General]
As of writing, Tero Subtitler is (or might be) missing support for the following subtitle formats, so samples and specifications would help.

- Adobe Premiere CC Basic Timed Text
- Amazon Prime
- Brightcove Player
- DECE CFF
- DFXP
- Disney IMSC 1.1
- [IMSC 1.1](https://www.w3.org/TR/ttml-imsc1.1)
- Kaltura Player
- NBC Universal Timed Text
- [Netflix IMSC 1.1](https://partnerhelp.netflixstudios.com/hc/en-us/articles/360053755033-Netflix-IMSC-1-1-Text-Profile)
- NCAM Multi-Language
- Ooyala Timed Text
- Red Bull Media Timed Text
- Scenarist (SCC)
- Sony Pictures Timed Text

Are there others?

**Comment by chenlung** [comment]
Tero Subtitler probably lacks support for formats using colours (e.g. EBU STL, ASS).

**Reply by mike-rider** [reply]
Just came across the app and am really liking it. Is this still being developed? Would love for scc support to be added. I can supply some sample files if needed.

**Reply by chenlung** [reply]
It’s on something of a hiatus until @URUWorks returns to it. 😊

**Comment by tilmannroe** [comment]
The SMPTE 428-7-2014 D-cinema standard is actually quite important for everyone working with projects that will have cinema distribution. It is necessary for DCP and DCDM creation. All paid, "professional" subtitle tools do support it, as well as SubtitleEdit on Windows.
The specifications can be found here:
https://pub.smpte.org/doc/st428-7/20140211-pub/
https://pub.smpte.org/doc/st429-12/20081203-pub/
